uber/server.py

```diff
@@ -60,11 +60,7 @@ def common_js(self):
 
     static_views = StaticViews()
 
-_sections = [path.split('/')[-1][:-3] for path in glob(os.path.join(c.MODULE_ROOT, 'site_sections', '*.py'))
-                                      if not path.endswith('__init__.py')]
-for _section in _sections:
-    _module = importlib.import_module('uber.site_sections.' + _section)
-    setattr(Root, _section, _module.Root())
+mount_site_sections(c.MODULE_ROOT)
```

Comment:
This function doesn't support modifying existing site sections.  Although that wouldn't be difficult to add.  Theoretically we could make it behave just like the ``mixin`` decorators, so like if you had a ``site_sections/registration.py`` file then it would look at the ``Root`` class in that file and use it to just override whatever was already in the ``Root`` class of the core plugin's ``site_sections/registration.py`` file.

We haven't had a need for that yet, but it sounds like something that would probably end up being useful one of these days.